### PR TITLE
rephrases second sentence in first assignment in Ruby/Test Driven Development lesson

### PR DESCRIPTION
**Pull request by GitHubBlues**
- [ x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x ] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [x ] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
- [ x] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1.Describe the changes made and include why they are necessary or important:

The first assignment of the Ruby/Test Driven Development lesson indicated that the reading assignment was about code testing in particular, but that it also applied to test driven development (TDD). This is not true as the text deals specifically with TDD (as mentioned in the third paragraph of the linked text). 
The second sentence in Odins curriculum was thus changed, so that it is clear that the reading assignment is about TDD in particular, but that it also applies to testing in general. 

This was discussed with rlmoser on discord #contributions-suggestions-bugs-discussions today (11-19-2021 at 4:51 AM)

#### 2. Related Issue

keyword #XXXXX

### CODE REVIEW

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
Looks like you branched off of your Knowledge Check PR and include the commit from it. You'll either need to remove that commit (I can help walk you through that if you like, it's a bit sketchy of a process when you're not used to it) or just wait until that other PR is merged before this one can be.

**Comment by GitHubBlues** [comment]
I do not really know what you mean. But its OK for me to wait for the other PR ... Thanks for your comment.

**Review by wise-king-sullyman — APPROVED** [review]
This should be good to go now! Thanks!

**Pull request merged**
